database/migrations/sqls/20240529120605-pgwar-entity-up.sql

```diff
@@ -20,6 +20,7 @@ CREATE TABLE IF NOT EXISTS pgwar.entity_preview(
     last_second bigint,
     parent_classes jsonb,
     ancestor_classes jsonb,
+    fk_class_modified timestamp with time zone,
```

Comment:
I suggest to rename the column fk_class_modified as all "fk_" columns are usually actual foreign key columns.
tmsp_fk_class_modification should be more explicit.